Wireframe/css/style.css

```diff
@@ -0,0 +1,96 @@
+  body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+  }
+  
+  header {
+    background-color: #333;
+    color: white;
+    text-align: center;
+    padding: 20px;
+  }
+  
+  main {
+    padding: 20px;
+  }
+
+  h1 {
+    margin: 0;
+    font-size: 2.5em;
+  }
+  
+  .articles {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+  
+  .articles > article:first-child {
+    padding: 20px;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 50px;
+  }
+  
+  .articles > article:first-child img {
+    width: 600px;
+    height: auto;
+    border-radius: 5px;
+    margin-bottom: 15px;
+    align-self: center;
+  }
+  
+  .article-grid {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    margin-bottom: 50px;
+  }
+  
+  article {
+    background-color: #dee2e6;
+    padding: 20px;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 50px;
+  }
+  
+  article img {
+    width: 350px;
```

Comment:
What would happen if your screen was 320px wide? An iPhone SE is this size. 

Turn on "devices" in Devtools and take a look https://developer.chrome.com/docs/devtools/device-mode